### PR TITLE
Add initial implementation for disable/enable Azure ops and other fixes

### PR DESCRIPTION
**Pull request by milismsft**
Add initial implementation for disable/enable ops; we will revisit these when moving to use AppGateway; for the time being we will power down/restart the instances only.
Return proper create time and image name when querying server group details.
Other minor fixes.

### CODE REVIEW

**Comment by milismsft** [comment]
@spinnaker/netflix-reviewers PTAL

**Comment by milismsft** [comment]
@spinnaker/netflix-reviewers  ping

File: clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/servergroup/ops/DisableAzureServerGroupAtomicOperation.groovy [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  /**
+   * curl -X POST -H "Content-Type: application/json" -d '[ { "disableServerGroup": { "serverGroupName": "taz-web1-d1-v000", "name": "taz-web1-d1-v000", "account" : "azure-cred1", "cloudProvider" : "azure", "appName" : "taz", "regions": ["westus"], "credentials": "azure-cred1" }} ]' localhost:7002/ops
+   */
+  @Override
+  Void operate(List priorOutputs) {
+    task.updateStatus BASE_PHASE, "Initializing Disable Azure Server Group Operation..."
+    for (region in description.regions) {
+      if (description.serverGroupName) description.name = description.serverGroupName
+      if (!description.application) description.application = description.appName ?: Names.parseName(description.name).app
+      task.updateStatus BASE_PHASE, "Disablinging server group ${description.name} " + "in ${region}..."
```
Comment (duftler) [inline clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/servergroup/ops/DisableAzureServerGroupAtomicOperation.groovy]: s/Disablinging/Disabling

**Comment by duftler** [comment]
LGTM.
Spotted one typo and a handle of new files with 2015 copyrights. Feel free to fix those in a later PR.

**Pull request merged**